### PR TITLE
Convert kisses from do_after to cooldown

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

- Rewrites https://github.com/Bubberstation/Bubberstation/pull/2045 to use cooldowns instead of a do_after timer, include an early return, and use modern balloon alerts instead of chat log messages.

- Sets the delay for harmful kisses at 1.6 seconds (allowing a player to cause a maximum ~25 damage per second using kisses.)

Thanks @Swiftfeather for the original PR allowing kiss of death to remain.

## Why It's Good For The Game

- A 0.55s cooldown on harmful kisses is ~63 damage per second, allowing someone to crit a player near instantly. Likely why TG had it set at 3 seconds. Given the increased health on SR/Bubber, 1.6 seems a better middle ground.

- do_after style timer on a single emote throws off players who have an expectation and muscle memory about how emotes 'act.' Switching it to a cooldown maintains the same emote feel for a kiss as all the others.

## Proof Of Testing

https://github.com/user-attachments/assets/47f4983a-f450-4fdb-8010-9e5e339926c2

## Changelog

:cl: LT3
code: Boykissers resume boy kissing! Kiss emote no longer causes a progress bar after using
balance: Syndie/death kiss projectiles have a 1.6 second cooldown
/:cl:

### CODE REVIEW

**Comment by Erol509** [comment]
Yep, normal kiss machinegunning shall die.

**Comment by lessthnthree** [comment]
Okay I'll keep a cooldown on normal kisses too.

File: modular_zubbers/code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
-			kiss_type = /obj/item/hand_item/kisser/syndie
+	if(HAS_TRAIT(user, TRAIT_SYNDIE_KISS))
+		kiss_type = /obj/item/hand_item/kisser/syndie
+		kiss_cooldown = KISS_COOLDOWN * 4
 
-		if(HAS_TRAIT(user, TRAIT_KISS_OF_DEATH))
-			kiss_type = /obj/item/hand_item/kisser/death
+	if(HAS_TRAIT(user, TRAIT_KISS_OF_DEATH))
+		kiss_type = /obj/item/hand_item/kisser/death
+		kiss_cooldown = KISS_COOLDOWN * 4
```
Comment (StrangeWeirdKitten) [inline modular_zubbers/code/modules/mob/living/emote.dm:21]: Is there a reason why we're making the kiss of death take 1.6 seconds?

Comment (StrangeWeirdKitten) [reply modular_zubbers/code/modules/mob/living/emote.dm:21]: Nevermind, I see the resoning in the PR.

**Pull request merged**